### PR TITLE
[MB-2005] Release 6.1.0

### PR DESCRIPTION
**Pull request by hcrowell**
Version 6.1.0
 - Updated iOS iOS Urban Airship SDK to 8.0.2
 - Added support for enabling/disabling the "displayASAPEnabled" property for in-app messages

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,9 @@
+===========================
+Version 6.0.2 - Oct 4, 2016
```
Comment (rlepinski) [inline CHANGELOG]: 6.1.0

**Comment by rlepinski** [comment]
👍 after bumping version to 6.1.0

**Comment by z0br0wn** [comment]
😎

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,9 @@
+===========================
+Version 6.1.0 - Oct 4, 2016
+===========================
+ - Updated iOS iOS Urban Airship SDK to 8.0.2
```
Comment (z0br0wn) [inline CHANGELOG]: iOS x2

**Pull request merged**